### PR TITLE
Sample app

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: modules/hello-world/pages/sample-application.adoc [hunk 1]
```diff
@@ -1,9 +1,180 @@
...
 :page-aliases: ROOT:sample-application,ROOT:tutorial4,ROOT:sample-app-backend
 
+[abstract]
+Discover how to program interactions with the Couchbase Server via the data, Query, and search services -- using the Travel Sample Application with the built-in Travel Sample data Bucket.
+This version also demonstrated the Developer Preview of the upcoming Collections feature.
 
-include::ref:partial$beta-warning.adoc[]
+6.5@sdk:pages:partial$sample-application.adoc
+
+This version of the Travel Sample App uses the xref:6.5@server:developer-preview:preview-mode.adoc[Collections Developer Preview (DP)]
```
Comment (daschl) [inline modules/hello-world/pages/sample-application.adoc:13]: if this is a decision that has been agreed on widely I am okay with it too

File: modules/hello-world/pages/sample-application.adoc [hunk 2]
```diff
@@ -1,9 +1,180 @@
...
+
+[source,bash]
+.Enable Collections Developer Preview
+----
+/opt/couchbase/bin/couchbase-cli enable-developer-preview --enable -c http://localhost:8091 -u Administrator -p password
+----
++
+[source,bash]
+----
+Developer preview cannot be disabled once it is enabled. If you enter developer preview mode you will not be able to upgrade. DO NOT USE IN PRODUCTION.
```
Comment (daschl) [inline modules/hello-world/pages/sample-application.adoc:44]: this is the third time it is mentioned in quick succession - maybe this one tip is enough?

File: modules/hello-world/pages/sample-application.adoc [hunk 3]
```diff
@@ -1,9 +1,180 @@
...
+Are you sure [y/n]: y
+----
+
+The Travel Sample Bucket needs altering to be split into collections. 
+There is a script to do this included with the Travel Sample App -- run:
+
+[source,bash]
+.Create Sample Collections
+----
+sh create-collections.sh Administrator password 127.0.0.1
```
Comment (daschl) [inline modules/hello-world/pages/sample-application.adoc:54]: what if people are on windows?

File: modules/hello-world/pages/sample-application.adoc [hunk 4]
```diff
@@ -1,9 +1,180 @@
...
+
+[source,bash]
+----
+mvn spring-boot:run
+----
+
+Most likely, you'll want to open up your preferred IDE for the `storage.host` step, 
+and stay there to build the app, rather than running Maven from the command line.
+
+After your preferred activity away from the screen for a couple of minutes of build
```
Comment (daschl) [inline modules/hello-world/pages/sample-application.adoc]: is it really taking minutes?

**Review by daschl — CHANGES_REQUESTED** [review]

File: modules/hello-world/pages/sample-application.adoc (hunk 4 above)
Comment (RichardSmedley) [reply modules/hello-world/pages/sample-application.adoc]: On my network, yes, we can't all live in the 21st Century you know :-(

File: modules/hello-world/pages/sample-application.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/hello-world/pages/sample-application.adoc:13]: To be revisited post-GA

File: modules/hello-world/pages/sample-application.adoc (hunk 3 above)
Comment (RichardSmedley) [reply modules/hello-world/pages/sample-application.adoc:54]: No reason for it not to work on Powershell, or _Bash for Windows_

**Review by daschl — APPROVED** [review]

**Pull request merged**